src/subscribables/observable.js

```diff
@@ -25,8 +25,13 @@ ko.observable = function (initialValue) {
 
     if (DEBUG) observable._latestValue = _latestValue;
     observable.peek = function() { return _latestValue };
-    observable.valueHasMutated = function () { observable["notifySubscribers"](_latestValue); }
-    observable.valueWillMutate = function () { observable["notifySubscribers"](_latestValue, "beforeChange"); }
+    observable.valueHasMutated = function () {
+        observable.updateVersion();
```

Comment:
I suppose we could update the version in both cases (mutation and notification), at least for rate-limited and sleeping observables.